contracts/src/v0.7/dev/UpkeepTranscoderInterface.sol

```diff
@@ -1,11 +0,0 @@
-// SPDX-License-Identifier: MIT
-
-pragma solidity ^0.8.0;
-
-interface UpkeepTranscoderInterface {
```

Comment:
This existed in v0.7 and v0.8, but it's pragma was v0.8 in both so it wasn't actually usable in v0.7. I think we should consolidate on one version for now and copy it backwards once the API is finalized.